BitwardenShared/UI/Auth/Login/LoginProcessor.swift

```diff
@@ -159,6 +159,8 @@ class LoginProcessor: StateProcessor<LoginState, LoginAction, LoginEffect> {
                 coordinator.navigate(
                     to: .twoFactor(state.username, .password(state.masterPassword), authMethodsData, nil)
                 )
+            case .twoFactorProvidersNotConfigured:
+                await handleErrorResponse(error)
```

Comment:
⚠️ Either you handle the error (and it's logging here) or in the service. Now it's going to log the error twice, once in the service and once here. If you want just remove the check in the service and leave this one since you have a specific error case it's going to be easy to pin point the origin of the error.

Reply:
I chose to handle the error here and remove the check in the service.